libsspl_sec/tests/acceptance/helpers/base64.c

```diff
@@ -25,7 +25,7 @@
 #include <math.h>
 #include <openssl/bio.h>
 #include <openssl/evp.h>
-BIO_METHOD* BIO_f_base64(void);
+// const BIO_METHOD* BIO_f_base64(void);
```

Comment:
so build not failing without this function now. openssl restricting here, check my comment [here](https://github.com/Seagate/cortx-monitor/pull/463#discussion_r676623244)

Reply:
no build doesn't fail, since the definition is already present in openssl/evp.h

Reply:
Ok so we dont need this explicit declaration now ?